rotkehlchen/api/v1/schemas.py

```diff
@@ -3127,6 +3127,14 @@ def transform_data(
         else:
             address = data['address']
 
+        try:
+            AddressbookEntry.check_chain_ecosystem(address)
+        except AddressNotSupported as e:
+            raise ValidationError(
+                'Given address is from a non supported ecosystem',
```

Comment:
```suggestion
                'Given address is from an unsupported ecosystem',
```
I think this sounds a bit better.